docs/Configurations.md

```diff
@@ -1,99 +1,86 @@
-
-    - Clone the mock firebolt OS repository` https://github.com/rdkcentral/mock-firebolt`
-    - Start the mock Firebolt OS based on steps mentioned in the repository.
-    - Start the Firebolt Certification App with `npm start`
-    - On launch add the `mf=true` queryParameter to the app launch url and relaunch the app.
-    The above steps will by default connect the Firebolt Certification App to the mock Firebolt OS running on localhost at port 9998
-    Note: To change the hosted location of the mock Firebolt OS use the guidelines as mentioned in `https://github.com/rdkcentral/mock-firebolt/blob/main/docs/UsageWithinApps.md#activating-mock-firebolt
-```
-
 ### Background
 
-TODO: add details
+Connecting to Mock Firebolt enables testing against a simulated Firebolt OS, providing a safe and flexible enviroment for development and testing.
 
 ### Setup
 
-TODO: add details
+- Clone the Repository [Mock-Firebolt](https://github.com/rdkcentral/mock-firebolt)
+- Follow the instructions provided in the repository to start the Mock Firebolt.
+- Launch the Firebolt Cretification App and add `mf=true` to the app launch URL to connect to the mock environment.
 
 ### Usage
 
-TODO: add details
+- Use Mock Firebolt for testing API integrations and app behavior in a simulated Firebolt environment.
+
 
 ## Supported URL parameters
- 
-EXISTING CONTENTS(needs revision):
-```
-- Platform: platform=`<platform>`
-  - The supported TARGET values are passed. While executing the test suite, if we provide wrong platform it will not execute the suite and will show "Unsupported target used." error
-- Lifecycle Validation: lifecycle_validation=true
-  - When we give lifecycle_validation=true it blocks the default execution of `lifecycle.ready` and `lifecycle.finished` method.
-  - This will help us to validate lifecycle API's as per our need
-- MFOS: mf=true
-  - When we are passing mf=true or with userId, FCA will connect to MFOS server and when we invoke any api in FCA it will return the response from MFOS.
-- System Ui: systemui=true
-  - If FCA systemui=true, FCA acts as the base app in case of ripple. The background color will be changed to purple, and it will display one more button as "Launch FCA app" to launch FCA as third-party app on Ripple devices.
-- TestContext: testContext=true
-  - If testContext=true, it will add the field context in mocha report generated
-```
+
 
 ### Background
 
-TODO: add details
+URL parameters allow customization of the FCA's behavior, enabling testers to tailor the testing environemt to specific needs.
 
 ### Setup
 
-TODO: add details
+- Use various parameters like 'platform', 'lifecycle_validation','mf','systemui' and more as per the requirment.
+- These parameters modify how the FCA interacts with the environemtn and the test suite.
 
 ### Usage
 
-TODO: add details
+Customize the test execution and reporting based on the provided URL parameters for targeted testing scenarios.
+
+**Parameters usage:**
```

Comment:
Can this be a table with columns
Parameter | Description | Possible Values | and maybe Notes?

Reply:
This looks great. Just a few notes:
- `platform` I'm not understanding what this would be, what are values we use?
- mf: `affects how APIs in FCA respond.` using MF means, based on the configuration, it calls method on MF instead of the device right? Think we can say that more clearly.
- systemui: changing the color is just a visual queue for users right? How does it actually change the FCA behavior?
- reportingId: does it append it or replace the uuid with the passed one?
- standalone: Let's link to the standalone doc in intentReader for more usage instructions (not merged into the main doc branch yet but can still create the hyperlink): https://github.com/rdkcentral/firebolt-certification-app/blob/FIRECERT-1500/docs/IntentReader.md#standalone

Wondering if some of these params require more info on how to use them. Not necessarily all of them, for example `voiceGuidance` is pretty self explanatory, but `testContext` I'm wondering what is the additional context provided in the report? Is that based on a config or user input? Or is it a static thing added when true?
Basically wondering if you turn on any of these url params is there additional steps that need to be taken by users?

Reply:
Looks good, just two small notes:
- Seems like the `Description` and `Notes` for system ui feels like we repeat info between the two columns.
- In reportingID `Description` need a space after URL.

Reply:
done